os/include/tinyara/pm/pm.h

```diff
@@ -200,7 +200,7 @@
 #endif
```

Comment:
Lets prefer not to have a product based commit description
You can indicate via the description that the reason is to lower the threshold from the current total transition time of 15 seconds which is too much to wait for.